packages/frontend/core/src/modules/docs-search/entities/docs-indexer.ts

```diff
@@ -36,7 +36,7 @@ export class DocsIndexer extends Entity {
   /**
    * increase this number to re-index all docs
    */
-  static INDEXER_VERSION = 2;
```

Comment:
Gabe, is that you?

Reply:
?

Reply:
I saw 3 is skipped.

![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/T2klNLEk0wxLh4NRDzhk/534f7a86-d6a0-4afc-bc45-a0ed0b673a99.png)